CSS/style.css

```diff
@@ -0,0 +1,297 @@
+/* Mobile first */
+
+@media screen and (min-width: 480px) {
+
+}
+
+
+/* {
+    border: 1px solid #999;
+}*/
+
+body {
+    color: #2d2d2d;
+    font-family: "Raleway", sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+}
+
+ol, ul {
```

Comment:
Becareful to not use selectors that are too general. These styles will be applied globally.